Runtimes/NET/BepInEx.NET.Common/BepInEx.NET.Common.csproj

```diff
@@ -12,6 +12,18 @@
 
     <ItemGroup>
         <ProjectReference Include="..\..\..\BepInEx.Core\BepInEx.Core.csproj"/>
+        <ProjectReference Include="..\..\..\BepInEx.PluginProvider\BepInEx.PluginProvider.csproj">
+            <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+            <OutputItemType>Content</OutputItemType>
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Targets>Build;DocumentationProjectOutputGroup</Targets>
+        </ProjectReference>
+        <ProjectReference Include="..\..\..\BepInEx.PatcherProvider\BepInEx.PatcherProvider.csproj">
+            <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+            <OutputItemType>Content</OutputItemType>
+            <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+            <Targets>Build;DocumentationProjectOutputGroup</Targets>
```

Comment:
Are these intended?

Reply:
Yes, it's to include the csproj, but not actually reference it so you still get it in the build output, but no one depends on it.